index.js

```diff
@@ -75,7 +75,7 @@ Jung.prototype.execute = function Jung$execute(triggerFile) {
     self.child = spawn(
         command[0]
       , command.slice(1)
-      , {env: env, cwd: process.cwd()}
+      , {env: env, cwd: process.cwd(), stdio: 'inherit'}
```

Comment:
this should only go in if `quiet` isn't specified.

also you need to remove the old lines around L83